class/practice_2.2/test.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
```

Comment:
It is better to have a set of files as test case prerequisite so there is no need to hard code it here. Just deliver test case together with attached drop-ins. Otherwise it will be difficult to test on bigger images resolution.